pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -631,11 +627,7 @@ impl RemoteTimelineClient {
             upload_queue.latest_files_changes_since_metadata_upload_scheduled,
         );
 
-        let index_part = IndexPart::new(
-            upload_queue.latest_files.clone(),
-            disk_consistent_lsn,
-            metadata,
-        );
```

Comment:
Luckily we never used this with something else than the `upload_queue.latest_metadata.clone()` because then we would had read the disk consistent lsn from elsewhere :)